src/documents/ComponentConfigurationOptions.md

```diff
@@ -468,19 +446,51 @@ Component developers are free to define their own additional options.
   </tr>
 </table>
 
+### `styles` ###
+<table>
+  <tr>
+    <th>Description</th>
+    <td>A record containing named CSS classes that the component will apply to its markup in order to achieve state-dependent styling effects.</td>
+  </tr>
+  <tr>
+    <th>Notes</th>
+    <td>The contents of this block are not interpreted by the framework at all. The existence of this block amounts to a helpful convention that implementors of view components are recommended to use, to organise
+    and advertise the CSS class names that they will apply on behalf of their users</td>
+  </tr>
+  <tr>
+    <th>Example Definition</th>
+    <td><pre>
+fluid.defaults("demo.initGridReorderer", {
```

Comment:
Could you explain what you are seeing here? It appears ok to me: https://github.com/amb26/infusion-docs/blob/FLUID-5249/src/documents/ComponentConfigurationOptions.md